packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -190,7 +190,8 @@ export const taggingWiring = createWiring({
     storeClickedResultWire
   },
   UserClickedResultAddToCart: {
-    trackAddToCartWire
+    trackAddToCartWire,
+    trackResultClickedWire
   },
   UserClickedPDPAddToCart: {
     trackAddToCartFromSessionStorage
```

Comment:
If we want to track the add2cart click on PDP we should also add the wire here

Reply:
Then i don't understand why the test about PDP tragging is passing...